### ISSUE TITLE
Editor should not be printed in Article type

### ISSUE THREAD

**Issue by junoslukan** [plk/biblatex-apa#230]
In Article type, the editor is printed as follows:

> Author, A., & Bauthor, B. (2024). Article title (E. Editor, Ed.). Journal name, Vol(Iss).

I do not think this should be the case according to [APA examples](https://apastyle.apa.org/style-grammar-guidelines/references/examples/journal-article-references). Editors are only mentioned in retracted articles.

**Comment by plk** [comment]
Why does an article have an editor field at all?

**Comment by junoslukan** [comment]
I am not completely sure, but I stumbled upon this with PLOS ONE that has hundreds of editors, so this information might be relevant. Similarly, it might be used for articles published in special issues that, again, can have other editors than the journal's regular issues.

BiBLaTeX's manual only has this to say (under article on p. 9):

> Note that editor and related fields refer to the journal while translator and related fields refer to the article.

**Comment by plk** [comment]
I don't think this is much of an issue as no-one has mentioned this before in many year of APA style usage. It is always possible to strip out editors from all articles dynamically using a sourcemap without chaning the `.bib` data if necessary.
